acronym-mapper/src/main/java/de/intension/acronym/AcronymUtil.java

```diff
@@ -20,4 +35,15 @@ private static String shorten(String value) {
         }
         return value.length() <= 2 ? value : value.substring(0, 2);
     }
+
+    private static String firstCharToUpperCase(String value)
```

Comment:
maybe add JavaDoc or similar (`@NotNull`), that `value` must not be **`null`** here? it is already given by using shorten() on the value first, but maybe better safe than sorry for future development?

Reply:
if it is "OK", I would keep it as it is to avoid unnecessary checks inside a private method.

Reply:
it is well OK, but a note in JavaDoc or similar could be helpful

Reply:
ok, @NotNull added to the parameter